pages/index.js

```diff
@@ -130,19 +176,27 @@ class Index extends ListPage {
   };
 
   renderList = () => {
+    const { localEditorHelperList } = this.state;
     const { articles = null, totalCount, authFields } = this.props;
     return (
       <div>
         <p>{totalCount} articles</p>
         {this.renderPagination()}
         <ul className="article-list">
-          {articles.map(article => (
-            <ArticleItem
-              key={article.get('id')}
-              article={article}
-              requestedForReply={authFields.get(article.get('id'))}
-            />
-          ))}
+          {articles.map(article => {
+            const id = article.get('id');
+            const EditorHelperState = localEditorHelperList[id];
```

Comment:
Since it's not a class, let's keep it lower-cased.
Or maybe just `{...localEditorHelperList[id]}` in line 196, so that we don't need a variable at all